x/ibc/07-tendermint/types/client_state.go

```diff
@@ -368,59 +380,59 @@ func (cs ClientState) VerifyNextSequenceRecv(
 
 	bz := sdk.Uint64ToBigEndian(nextSequenceRecv)
 
-	if err := proof.VerifyMembership(consensusState.GetRoot(), path, bz); err != nil {
+	if err := merkleProof.VerifyMembership(consensusState.GetRoot(), path, bz); err != nil {
 		return sdkerrors.Wrap(clienttypes.ErrFailedNextSeqRecvVerification, err.Error())
 	}
 
 	return nil
 }
 
-// validateVerificationArgs perfoms the basic checks on the arguments that are
+// sanitizeVerificationArgs perfoms the basic checks on the arguments that are
 // shared between the verification functions.
-func validateVerificationArgs(
+func sanitizeVerificationArgs(
```

Comment:
I thought this would be better naming if I was returning a commitmenttypes.MerkleProof. I unmarshal it here because otherwise it would be redundant code among each verify func